client_packages/LSOnline/Vehicle/Vehicle.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+// Main render for vehicles
+mp.events.add('render', () =>
+{
+	const controls = mp.game.controls;
+	
+	controls.enableControlAction(0, 23, true);
```

Comment:
extract rendering to another class as a single function. add VehicleManager instead of Vehicle.js(this name can suggest that this is a model file).

Reply:
Renamed Vehicle.js to VehicleRender.js as main render for vehicles.